### PR TITLE
adding deploy

### PR DESCRIPTION
**Pull request by caseyjlaw**

### CODE REVIEW

File: deploy [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
```
Comment (rh-codebase) [inline deploy]: hopefully this script is run via a service so not sure how useful this is unless the service know what to do with it. The group should decide if this feature is needed for the service.

File: deploy [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+echo $HOSTNAME  # should be installing on calim
```
Comment (rh-codebase) [inline deploy]: this should directed to log file

File: deploy [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+echo $PWD  # should be base of code repo
+whoami  # should be user pipeline (or ubuntu?)
```
Comment (rh-codebase) [inline deploy]: add checks for proper user and proper directory.

File: deploy [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+# setup
+DATE=$(date --iso-8601=seconds)
+LOG=~/deploy_lwa-antpos_$DATE.log
+
+if [[ $HOSTNAME =~ 'calim' ]]; then
+    echo "$DATE" >> $LOG 2>>$LOG
```
Comment (rh-codebase) [inline deploy]: the group needs to discuss how growing log files should be managed. If it's the script's responsibility then the script needs logic to either rotate log files or truncate them.

File: deploy [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    conda run -n datarecorder which python >> $LOG 2>>$LOG
+    conda run -n datarecorder git describe >> $LOG 2>>$LOG
+    conda run -n datarecorder pip install . >> $LOG 2>>$LOG
+else
+    echo "Not installing on this host"
```
Comment (rh-codebase) [inline deploy]: to log file.

File: deploy [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+echo $PWD  # should be base of code repo
+whoami  # should be user pipeline (or ubuntu?)
+
+# setup
+DATE=$(date --iso-8601=seconds)
+LOG=~/deploy_lwa-antpos_$DATE.log
+
+if [[ $HOSTNAME =~ 'calim' ]]; then
+    echo "$DATE" >> $LOG 2>>$LOG
+    conda run -n datarecorder which python >> $LOG 2>>$LOG
```
Comment (rh-codebase) [inline deploy]: check $? after each command for non-zero error codes.

File: deploy [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    conda run -n datarecorder git describe >> $LOG 2>>$LOG
+    conda run -n datarecorder pip install . >> $LOG 2>>$LOG
+else
+    echo "Not installing on this host"
+fi
```
Comment (rh-codebase) [inline deploy:26]: i'd like to see: git describe logged

File: deploy (hunk 1 above)
Comment (caseyjlaw) [reply deploy]: I say we keep it for potential testing or manual use.

**Pull request merged**